app/experimenter/nimbus-ui/src/components/ConfidenceInterval/index.tsx

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* istanbul ignore file */
```

Comment:
@jodyheavener Thanks for this, it totally slipped my mind I could add this. I left the `TableVisualizationRow` file as-is though, it ended up being more effort than that one was worth adding `/* istanbul ignore next */` everywhere and I don't want us to miss removing them later.